README.md

```diff
@@ -22,6 +22,33 @@ What can I do?
 - Add this [badge](externals/instruction.md) to your project to support 996.ICU.  
 把这个[徽章](externals/instruction.md)加入你的项目来支持996.ICU
 
+
+The principle and purpose Of this repository!
+---
+
+* this is not a politic activity,we are the staunch supporters of labour law. we ask employers to respect the legitimate rights and interests of employees.
```

Comment:
This is not a political activity, we are the staunch supporters of labour law. We ask employers to respect the legitimate rights and interests of employees.